.env.example

```diff
@@ -1,3 +1,4 @@
 BOT_TOKEN=your bot token
 CONFIG_EDITING_USERS=["502575111346323456", "229380062909038594", "119922034812780547", "267678060914933770", "170429173003714560", "245958554899578880"]
-CONFIG_EDITING_GROUPS=["737810664034271302"]
\ No newline at end of file
+CONFIG_EDITING_GROUPS=["737810664034271302"]
+DISCORD_UPDATE_CHANNELS=["349264069007638530"]
```

Comment:
Shouldn't this be in the config as it's not sensitive or needed to edit the config itself if it goes wrong?